notebooks/astradb_haystack_integration.ipynb

```diff
@@ -0,0 +1,1112 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.    query_pipeline = Pipeline()
</pre><p>Ah this is intersting to see. Thechnically this pipeline is an "indexing" pipeline so I think you shouldn't name it query_pipeline. If you like, reach out to me and I can explain why :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
oh duh of course, I know that 🤦🏻‍♂️ will fix!